index.html

```diff
@@ -4,13 +4,234 @@
+          aria-label="toggle-button"
+        >
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </button>
+
+        <button class="x-button display-none">
+          <span> &#10005;</span>
+        </button>
+
+        <ul class="display-none display-flex-lg justify-con-btw nav-ul">
+          <li class="nav-ul-li">
+            <a href="about.html" class="nav-li-anchor">About</a>
+          </li>
+          <li class="nav-ul-li">
+            <a href="" class="nav-li-anchor">Program</a>
+          </li>
+          <li class="nav-ul-li"><a href="" class="nav-li-anchor">Join</a></li>
+          <li class="nav-ul-li">
+            <a href="" class="nav-li-anchor">Sponsors</a>
+          </li>
+        </ul>
+      </nav>
     </header>
+    <main class="main">
+      <section class="home-Page">
+        <div class="home-page-first-div">
+          <p class="home-page-title red-like-color">"Refugee Food Festival"</p>
+          <h1 class="home-page-header red-like-color">
+            A Blue Pacific voice for the United Nations Food Systems Summit
+          </h1>
+          <p class="article-paragraph">
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam
+            elementum erat eu fermentum accumsan. Mauris euismod diam lacus, vel
+            bibendum elit luctus et.
+          </p>
+
+          <p class="date-paragraph">2015.10.15 (THU) ~ 16(FRI)</p>
+          <p class="location-paragraph">
+            @ National Museum of Korea Art Center Nabi, and more
+          </p>
+        </div>
+
+        <div class="main-program-div">
+          <h2 class="text-align-center">Main Program</h2>
+
+          <div
+            class="d-flex f-direction-col main-program-wrapper-div justify-con-cnt-lg flex-d-row-lg"
+          >
+            <ul class="main-program-ul flex-d-col-lg d-flex j-content-btw">
+              <li class="main-program-ul-li">
+                <img
+                  class="lecture-image"
+                  src="./images/bowl.png"
+                  alt="speaker"
+                />
+              </li>
+              <li class="main-program-ul-li topic">Lecture</li>
+              <li class="main-program-ul-li">
+                listen to the speaker from various country about sharing and
+                opening
+              </li>
+            </ul>
+
+            <ul class="main-program-ul flex-d-col-lg d-flex j-content-btw">
+              <li class="main-program-ul-li">
+                <img
+                  class="lecture-image"
+                  src="./images/speaking-summit.png"
+                  alt="speaker"
+                />
+              </li>
+              <li class="main-program-ul-li topic">Lecture</li>
+              <li class="main-program-ul-li">
+                listen to the speaker from various country about sharing and
+                opening
+              </li>
+            </ul>
```

Comment:
- [ ] Please follow the [design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) and implement the required changes on your program section on both desktop and mobile version
![4](https://user-images.githubusercontent.com/47625407/157782111-03496647-fa13-45f2-8c06-07812ea412bb.png)

Reply:
Thanks for the review change made

Reply:
Thanks for the review change made